src/amulware.Graphics/Core/Color.cs

```diff
@@ -363,42 +363,42 @@ public Color(Color color, byte newAlpha)
         /// <returns>The constructed colour.</returns>
         public static Color FromHSVA(float h, float s, float v, byte a = 255)
         {
-            float chroma = v * s;
+            var chroma = v * s;
             h /= MathHelper.PiOver3;
-            float x = chroma * (1 - Math.Abs((h % 2) - 1));
-            float m = v - chroma;
+            var x = chroma * (1 - Math.Abs(h % 2 - 1));
```

Comment:
```suggestion
            var x = chroma * (1 - Math.Abs((h % 2) - 1));
```
This is not entirely obvious to me.